.pre-commit-config.yaml

```
@@ -0,0 +1,10 @@
+repos:
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  # Ruff version.
+  rev: v0.12.9
+  hooks:
+    # Run the linter.
+    - id: ruff-check
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)